<launch>
	<node name="dvl" pkg="sensors" type="DVL">
		<!--<remap from="WH_DVL_data" to="vo"/>-->
	</node>
</launch>
